integrationtest/TargetProjects/NetCore/EmptyTestProject/Main.cs

```diff
@@ -0,0 +1,2 @@
+
+Console.WriteLine("this is a global statement");
```

Comment:
I think this should be in https://github.com/stryker-mutator/stryker-net/tree/master/integrationtest/TargetProjects/NetCore/TargetProject/Constructs as a new file called `Csharp9_TopLevelStatement.cs`. We are trying to keep an accurate representation of csharp features per language level so that we can easily figure out if we're missing certain constructs in the integration tests.

Reply:
Mhm now that I say that, is that even possibly or does your entry point have to be top-level as well for this to work?

Reply:
I think that should work. `An application must have only one entry point. A project can have only one file with top-level statements.` It doesn't state that it has to be a file in the root of the project or with a specific name.

Reply:
well
Now that I have tried to make the change, I know why I did not choose this route. Top level statements are allowed only for executable targets.
Shall I convert the project to output an executable instead of a library assembly?

Reply:
I think that's fine yes

Reply:
done